app/src/main/res/values/strings.xml

```diff
@@ -332,6 +332,7 @@
   <!-- AppVersionFragment -->
   <string name="app_version_name">App Version %s</string>
   <string name="app_last_update_date">The last update was installed on %s. Use the above version number to send feedback about bugs.</string>
+  <string name="label_app_version">App Version</string>
```

Comment:
It would be better to use
`app_version_activity_label`

Reply:
> It would be better to use
> `app_version_activity_label`

made the change thank